<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="cfgGroup">
     <property name="title">
      <string>CONFIGURATION</string>
     </property>
     <layout class="QGridLayout" name="cfgLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="addrValue"/>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="readButton">
        <property name="text">
         <string>Read cfg.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="writeButton">
        <property name="text">
         <string>Write cfg.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="hvGroup">
     <property name="title">
      <string>HIGH VOLTAGE</string>
     </property>
     <layout class="QGridLayout" name="hvLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="voltageLabel">
        <property name="text">
         <string>Voltage</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="voltageValue">
        <property name="maximum">
         <number>4095</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="voltageFeedback">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="currentLabel">
        <property name="text">
         <string>Current</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="currentValue">
        <property name="maximum">
         <number>4095</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="currentFeedback">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="stateLabel">
        <property name="text">
         <string>State</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="stateValue">
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="stateFeedback">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="discGroup">
     <property name="title">
      <string>DISCRIMINATOR</string>
     </property>
     <layout class="QGridLayout" name="discLayout"/>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="monoGroup">
     <property name="title">
      <string>MONOSTABLE</string>
     </property>
     <layout class="QGridLayout" name="monoLayout"/>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="coiGroup">
     <property name="title">
      <string>COINCIDENCE</string>
     </property>
     <layout class="QGridLayout" name="coiLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="cutLabel">
        <property name="text">
         <string>Number of hit detectors</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="cutValue">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
